### ISSUE TITLE
App crashes during photo display

### ISSUE THREAD

**Issue by sneJ-** [jellyfin/jellyfin-androidtv#3773]
### Describe the bug

Hello good people of this community,

 here is my bug description:

**Preperation:**
I created a media library for home videos and photos.
I added a top level directory to the library.
In the top level directory are two additional directories.
One for photos and one for videos.
In the video directory are around 50 videos.
In the photos directory are around 3100 videos of type .JPG.

**Bug reprodution:**
I start the Jellyfin app on my Fire TV stick.
I enter the top level library "Home Videos and Photos".
Both subdirectories photos and videos are displayed properly.
I enter the photos subdirectory.
All 3100 photos are dispayed properly as thumbnails.
I pick a random photo thumbnail to enlarge it.
The enlarged photo displays.
Now everything freezes and the Jellyfin app closes after 2-3 seconds.

**Further information**
I found following logs on my Jellyfin server that are mapped to the timestamps when I reproduced the bug.

[Log.txt](https://github.com/user-attachments/files/16319903/Log.txt)
[log2.txt](https://github.com/user-attachments/files/16319904/log2.txt)

I'm able to display the images properly on with my webbrowser Jellyfin app on my PC.

If you need any further information, please do not hesitate to contact me.

Best regards

Jens.

### Media info of the file

```shell
General
Complete name                            : C:\Users\Jens\Desktop\PICT0001.JPG
Format                                   : JPEG
File size                                : 1 023 KiB

Image
Format                                   : JPEG
Width                                    : 1 680 pixels
Height                                   : 2 520 pixels
Color space                              : YUV
Chroma subsampling                       : 4:2:2
Bit depth                                : 8 bits
Compression mode                         : Lossy
Stream size                              : 1 023 KiB (100%)
```

### Application version

0.16.11

### Device information

Amazon Fire TV Stick (older)

### Android version

5.1.1

### Jellyfin server version

10.9.7

**Comment by nielsvanvelzen** [comment]
Are you able to test if this issue still occurs in the 0.17 beta?

**Comment by sneJ-** [comment]
Hi,

I will be able to test if the issue still occurs in the 0.17 beta in around two weeks.
Then I'll be back at the household that have the old FireTV sticks.

I just tested with my newer FireTV 4k stick in my home and didn't have the issue.

**Comment by sneJ-** [comment]
Hi,

I was able to reproduce the issue on an old fire tv sticks with the yesterday's released 0.17 app version.

**Comment by nielsvanvelzen** [comment]
Please share the crash logs from the new app version.

**Comment by sneJ-** [comment]
Hi Niels,

here is the log.
[Log.txt](https://github.com/user-attachments/files/16488412/Log.txt)

I can also give some more information about the issue with the new version.
It only occurs if I hit the play/forward/revert button after opening the image within the first few seconds.

If I open the first image, wait like 20 seconds and hit then one of the three buttons everything works.

Therefore, I've now a workaround to provide my family with :-)

If you need any further information,

please do not hesitate to contact me.

Best regards

Jens

**Comment by nielsvanvelzen** [comment]
Thanks for the logs! It looks like the bug from 0.16 was indeed fixed, but another one appeared with 0.17. Fortunately, that one was quite easy to fix and will be included in the next release (0.17.1).

**Comment by sneJ-** [comment]
Thank you very much :-)
